tests/DevApps/Directory.Build.props

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFrameworks>net6.0; net7.0; net8.0</TargetFrameworks>
+    <TargetFrameworks Condition="'$(TargetNet9)'== 'True'">$(TargetFrameworks); net9.0</TargetFrameworks>
     <UseWip>true</UseWip>
     <IsPackable>false</IsPackable>
     <LangVersion>11</LangVersion>
```

Comment:
not this PR, but I thought we had moved everything to LangVersion 12 - fyi @kellyyangsong